charts/templates/externalsecret.yaml

```diff
@@ -0,0 +1,32 @@
+# Specify your own secret
```

Comment:
I believe this needs to be either configurable or removed

Reply:
I also made externalSecret configurable in `values.yaml`.